tokenserver-db-common/src/lib.rs

```diff
@@ -1,19 +1,27 @@
+#[macro_use]
+extern crate slog_scope;
+
 pub mod error;
 pub mod params;
 pub mod results;
 
-use std::time::Duration;
+use std::time::{Duration, SystemTime, UNIX_EPOCH};
 
 use async_trait::async_trait;
+use syncserver_common::Metrics;
 use syncserver_db_common::{GetPoolState, PoolState};
 
-use crate::error::DbError;
+pub use crate::error::{DbError, DbResult};
```

Comment:
I took a look at the other generic type aliases for `DbResult<T>` and those reside in the lib.rs modules for the syncstorage-mysql and syncstorage-spanner crates, so I think we should move this definition of `DbResult` into lib.rs here in tokenserver-db-common, or even tokenserver-db.

Reply:
:thumbsup: also made this error mod private